### ISSUE TITLE
Single test failure in two GCC builds only.

### ISSUE THREAD

**Issue by evoskuil** [libbitcoin/libbitcoin-system#1389]
```
unknown location(0): fatal error: in "chain_state_tests/chain_state__work_required_retarget__overflow_patch_disabled__lower_value": signal: illegal operand; address of failing instruction: 0x5628f21f6cb0
test/chain/chain_state.cpp(39): last checkpoint: "chain_state__work_required_retarget__overflow_patch_disabled__lower_value" test entry
```

**Comment by evoskuil** [comment]
https://github.com/libbitcoin/libbitcoin-system/actions/runs/7700901196/job/20985733213#step:9:28937

https://github.com/libbitcoin/libbitcoin-system/actions/runs/7700901196/job/20985733455#step:22:28

**Comment by evoskuil** [comment]
After removing this test another test failed, as this is not a test fail (false) but a process termination.

**Comment by evoskuil** [comment]
These test failures seem unrelated and inexplicable. Since they are only occurring in the two builds that are reporting as both DEBUG and NDEBUG we should resolve that issue first.

**Comment by pmienk** [comment]
Addressed by libbitcoin/libbitcoin-build#306 - The issue was unrelated to asserting NDEBUG while debug optimizing the build, but I have paired the debug optimization elsewhere.  The issue was having enabled avx512 on linux systems where it is not predictably present.  This was an error on my part, was testing privately and didn't revert changes before addressing a different issue.

**Comment by evoskuil** [comment]
Resolved
